src/mixer/basetrackplayer.cpp

```diff
@@ -335,3 +345,30 @@ void BaseTrackPlayerImpl::setupEqControls() {
     m_pSpeed = new ControlObjectSlave(group, "rate", this);
     m_pPitchAdjust = new ControlObjectSlave(group, "pitch_adjust", this);
 }
+
+void BaseTrackPlayerImpl::slotPassthroughEnabled(double v) {
+    bool configured = m_pInputEnabled->toBool();
```

Comment:
Yea, it's confusing -- here passthrough is being enabled, and we need to check that the input is configured. Whether it's configured is represented by an "enabled" control (for uniformity with aux and mic).